apinotes/CMakeLists.txt

```diff
@@ -1,24 +1,35 @@
-set(SWIFT_API_NOTES_INPUTS
-  Accelerate
-  Dispatch
-  ScriptingBridge
-  os
+set(sources
+  Accelerate.apinotes
+  Dispatch.apinotes
+  ScriptingBridge.apinotes
+  os.apinotes
 )
 
-if(NOT DEFINED SWIFT_API_NOTES_PATH)
-  message(FATAL_ERROR "Define SWIFT_API_NOTES_PATH before including this file")
-endif()
+set(output_dir "${SWIFTLIB_DIR}/apinotes")
 
-foreach(module ${SWIFT_API_NOTES_INPUTS})
-  if(NOT EXISTS "${SWIFT_API_NOTES_PATH}/${module}.apinotes")
-    message(SEND_ERROR "Missing apinotes for ${module}")
-  endif()
+add_custom_command(
+    OUTPUT "${output_dir}"
+    COMMAND ${CMAKE_COMMAND} "-E" "make_directory" "${output_dir}")
+set(outputs)
+foreach(input ${sources})
+  add_custom_command(
+      OUTPUT "${output_dir}/${input}"
+      DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/${input}"
+      COMMAND
+        "${CMAKE_COMMAND}" "-E" "copy_if_different"
+        "${CMAKE_CURRENT_SOURCE_DIR}/${input}"
+        "${output_dir}/${input}"
+      COMMENT "Copying ${input} to ${output_dir}")
+  list(APPEND outputs "${output_dir}/${input}")
 endforeach()
```

Comment:
I believe that we have a new enough CMake to avoid this.  Please use a single command (`${CMAKE_COMMAND_}  -E copy_if_different ${sources} ${output_dir}/`)

Reply:
This way allows CMake/Ninja to only copy the files that changed, but I guess `copy_if_different` will already do that, and N is very small.